__tests__/api.test.js

```diff
@@ -241,4 +270,129 @@ describe("PATCH /api/articles/:article_id/", () => {
         expect(res.body.msg).toBe("bad request");
       });
   });
+
+  test("PATCH:400 for invalid article ID format", () => {
+    return request(app)
+      .patch("/api/articles/not-a-number")
+      .send({ inc_votes: 1 })
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe("bad request");
+      });
+  });
+});
+
+describe("DELETE /api/comments/:comment_id", () => {
+  test("DELETE:204 deletes specific comment", () => {
+    return request(app)
+      .delete("/api/comments/10")
+      .expect(204)
+      .then((res) => {
+        const comment = [res.body];
+        expect(Object.keys(comment[0]).length).toBe(0);
```

Comment:
There is no need to explicitly check this as express by default will not send a request body if a 204 status code is sent back.